### PR TITLE
:octocat: Post deployment message to PR

### PR DESCRIPTION
**Pull request by neilbmclaughlin**

### CODE REVIEW

**Comment by neilbmclaughlin** [comment]
connecting-to-services pull request 251 deployed to connecting-to-services-pr-251.dev.c2s.nhschoices.net and using the latest release of nearby-services-api (0.2.0-beta)

**Comment by neilbmclaughlin** [comment]
connecting-to-services pull request 251 deployed to http://connecting-to-services-pr-251.dev.c2s.nhschoices.net and using the latest release of nearby-services-api (0.2.0-beta)

**Comment by neilbmclaughlin** [comment]
:rocket: connecting-to-services pull request 251 deployed to (http://connecting-to-services-pr-251.dev.c2s.nhschoices.net) and using the latest release of nearby-services-api (0.2.0-beta)

**Comment by neilbmclaughlin** [comment]
url (http:www.google.com)

**Comment by neilbmclaughlin** [comment]
url (http://www.google.com)

**Comment by neilbmclaughlin** [comment]
url (http://www.google.com)

**Comment by neilbmclaughlin** [comment]
url [here](http://www.google.com)

**Comment by neilbmclaughlin** [comment]
:rocket: [http://www.google.com](http://www.google.com)

**Comment by neilbmclaughlin** [comment]
:rocket: connecting-to-services pull request 251 deployed to [http://connecting-to-services-pr-251.dev.c2s.nhschoices.net](http://connecting-to-services-pr-251.dev.c2s.nhschoices.net) and using the latest release of nearby-services-api (0.2.0-beta)

**Comment by neilbmclaughlin** [comment]
Comment is attributed to me as Travis is using a personal access token I generated. Need to consider what we should use instead.

**Comment by neilbmclaughlin** [comment]
Need to remove Heroku integration.

**Comment by neilbmclaughlin** [comment]
Heroku review apps disabled for connecting to services.

**Comment by neilbmclaughlin** [comment]
:rocket: deployed to [http://connecting-to-services-pr-251.dev.c2s.nhschoices.net](http://connecting-to-services-pr-251.dev.c2s.nhschoices.net) and using the latest release of nearby-services-api (0.2.0-beta)

**Comment by neilbmclaughlin** [comment]
Issue raised regarding use of GitHub personal access tokens on Travis (#252)

File: scripts/rancher_pr_deploy.sh [hunk 1]
```diff
@@ -42,13 +42,12 @@ if [[ -n "$TRAVIS" ]]; then
 
     rancher -w up --pull --upgrade -d --stack "${RANCHER_STACK_NAME}" --env-file answers.txt
 
-    # SLACK NOTIFICATION
-    SLACK_MSG="connecting-to-services pull request ${TRAVIS_PULL_REQUEST} deployed to ${RANCHER_STACK_NAME}.dev.c2s.nhschoices.net and using the latest release of nearby-services-api (${LATEST_RELEASE})"
+    DEPLOY_URL="http://${RANCHER_STACK_NAME}.dev.c2s.nhschoices.net"
+    MSG=":rocket: deployed to [${DEPLOY_URL}](${DEPLOY_URL}) and using the latest release of nearby-services-api (${LATEST_RELEASE})"
```
Comment (st3v3nhunt) [inline scripts/rancher_pr_deploy.sh]: I guess the :shipit: emoji can be used when we deploy to production 😜

**Pull request merged**